examples/addr2line.rs

```diff
@@ -75,6 +87,23 @@ fn print_function(name: &str, language: Option<gimli::DwLang>, demangle: bool) {
     }
 }
 
+fn load_file_section<'input, 'arena, Endian: gimli::Endianity>(
+    id: gimli::SectionId,
+    file: &object::File<'input>,
+    endian: Endian,
+    arena_data: &'arena Arena<Cow<'input, [u8]>>
+) -> Result<gimli::EndianSlice<'arena, Endian>> {
+    // TODO: Unify with dwarfdump.rs in gimli.
+    let name = id.name();
+    let section = file.section_by_name(name).unwrap();
```

Comment:
Don't use `unwrap`, you need to handle the case where the section doesn't exist by returning an empty slice.